### PR TITLE
Docs: Small typo and proofreading edits

### PR DESCRIPTION
**Pull request by nicholassaylor**
## What is this fixing or adding?
Modifying game and set-up pages for typos, confusing/redundant sentences, and clarity.
No information was removed from these pages, and in some cases, clarifying information was added.

## How was this tested?
Looking at rendered markdown

### CODE REVIEW

File: worlds/kh2/docs/en_Kingdom Hearts 2.md [hunk 1]
```diff
@@ -37,10 +43,10 @@ It is added to your inventory. If you obtain magic, you will need to pause your
...
+When you die in vanilla Kingdom Hearts 2, you are reverted to the last non-boss room you entered and your status is reverted to what it was at that time. However, in archipelago, any item that you have sent/received will not be taken away from the player, any chest you have opened will remain open, and you will keep your level, but lose the experience.
 
 
-For example, if you are fighting Roxas and you receive Reflect Element and you die fighting Roxas, you will keep that reflect. You will still need to pause your game to have it show up in your inventory, then enter a new room for it to become properly usable.
+For example, if you are fighting Roxas, receive Reflect Element, then die mid-fight, you will keep Reflect Element. You will still need to pause your game to have it show up in your inventory, then enter a new room for it to become properly usable.
```
Comment (Silvris) [inline worlds/kh2/docs/en_Kingdom Hearts 2.md:49]:
```suggestion
For example, if you are fighting Roxas, receive Reflect Element, then die mid-fight, you will keep that Reflect Element. You will still need to pause your game to have it show up in your inventory, then enter a new room for it to become properly usable.
```

File: worlds/kh2/docs/en_Kingdom Hearts 2.md [hunk 2]
```diff
@@ -49,13 +55,12 @@ For example, if you are fighting Roxas and you receive Reflect Element and you d
...
-- Customize how many locations you want on Sora's Levels.
-- Customize the EXP Multiplier of everything that affects Sora.
-- Customize the Available Abilities on Keyblades.
-- Customize the level of Progressive Movement (Growth Abilities) you start with.
-- Customize the amount of Progressive Movement (Growth Abilities) you start with.
+- Customize how many World-Locking Items you need to progress in that world.
+- Customize the amount of World-Locking Items you start with.
+- Customize how many of Sora's Levels are locations.
+- Customize the EXP multiplier for Sora.
+- Customize the available bbilities on keyblades.
```
Comment (Silvris) [inline worlds/kh2/docs/en_Kingdom Hearts 2.md:62]:
```suggestion
- Customize the available abilities on keyblades.
```

File: worlds/kh2/docs/en_Kingdom Hearts 2.md [hunk 3]
```diff
@@ -49,13 +55,12 @@ For example, if you are fighting Roxas and you receive Reflect Element and you d
...
-- Customize the Amount of World Locking Items You Start With.
-- Customize how many locations you want on Sora's Levels.
-- Customize the EXP Multiplier of everything that affects Sora.
-- Customize the Available Abilities on Keyblades.
-- Customize the level of Progressive Movement (Growth Abilities) you start with.
-- Customize the amount of Progressive Movement (Growth Abilities) you start with.
+- Customize how many World-Locking Items you need to progress in that world.
+- Customize the amount of World-Locking Items you start with.
+- Customize how many of Sora's Levels are locations.
+- Customize the EXP multiplier for Sora.
```
Comment (Silvris) [inline worlds/kh2/docs/en_Kingdom Hearts 2.md:61]:
```suggestion
- Customize the EXP multiplier for Sora and his Drive Forms.
```
This still leaves out summon levels, but not sure there's a great way to word that.

**Review by Silvris — CHANGES_REQUESTED** [review]
Mostly a couple of nitpicks. The maintainers for Final Fantasy, KH2, and RoR2 should probably sign off on this before it gets merged, as these contained larger edits.

File: worlds/kh2/docs/en_Kingdom Hearts 2.md (hunk 3 above)
Comment (nicholassaylor) [reply worlds/kh2/docs/en_Kingdom Hearts 2.md]: Most suggestions were committed.
I tried adding mention of summons in commit ce0ea5d25bf6b53238f4cadc47e000e45bb0b6f4 , see if this is what you are looking for @Silvris .

**Comment by ThePhar** [comment]
Pinging code owners for worlds touched to review.

@Marechal-L for Dark Souls III
@jtoyoda for Final Fantasy
@JaredWeakStrike for Kingdom Hearts II
@KonoTyran for Minecraft
@kindasneaki for Risk of Rain 2
@agilbert1412 for Stardew Valley

**Review by agilbert1412 — APPROVED** [review]
I validated the Stardew Valley changes only

File: worlds/ror2/docs/en_Risk of Rain 2.md [hunk 4]
```diff
@@ -8,36 +8,38 @@ config file.
...
+  - Certain chests (made clear via a location check progress bar) will send an item out to the
+    multiworld. The location of these chests do not matter, since all environments share a unified location pool.
 
 Explore Mode:
 
-  - Just like in Classic mode chests will send out an item to the multiworld. The difference is that each environment
-  will have a set amount that can be sent out and shrines along with other things that will need to be checked.
-  Also, each environment is an item and, you'll need it to be able to access it.
+  - Chests will continue to work as they did in Classic Mode, the difference being that each environment
+  will have a set amount of items that can be sent out. In addition, shrines, radio scanners, newt shrines,
```
Comment (kindasneaki) [inline worlds/ror2/docs/en_Risk of Rain 2.md:25]:
```suggestion
  will have a set amount of items that can be sent out. In addition, shrines, radio scanners, newt alters, 
```

**Review by kindasneaki — CHANGES_REQUESTED** [review]
Just needs to fix the one change, and then it looks good to me

File: worlds/ror2/docs/en_Risk of Rain 2.md (hunk 4 above)
Comment (nicholassaylor) [reply worlds/ror2/docs/en_Risk of Rain 2.md]: 66517803864ed4881070b4cdead469838f609485

**Review by jtoyoda — APPROVED** [review]
Approved for FF1

**Review by kindasneaki — APPROVED** [review]

**Comment by KonoTyran** [comment]
@Marechal-L last ok we need to merge this.

**Comment by black-sliver** [comment]
@Br00ty or @Zunawe please check and approve/reject DS3

**Review by Zunawe — APPROVED** [review]
I think the changes are fine. As mentioned in Discord, the page could use a rewrite to better reflect the recent changes. I'll open a PR in a minute for that.

**Review by black-sliver — APPROVED** [review]
Looks like we got approvals from all involved games.

**Pull request merged**